packages/core-components/src/lib/atoms/inputs/text/TextInput.svelte

```diff
@@ -18,31 +17,30 @@
 	/** @type {string} */
 	export let name;
 
-	/**
-	 * @param {string} v
-	 */
-	const updateValue = debounce((v) => {
+	/** @type {boolean} */
+	export let unsafe = false;
```

Comment:
```suggestion
	export let unsafe = false;
	$: unsafe = unsafe === true || unsafe === 'true';
```

Reply:
I'm going to add this but I think we should point people towards just having `<TextInput unsafe/>` instead of `<TextInput unsafe=true/>`